apps/charterafrica/src/components/Datasets/DatasetCard.js

```diff
@@ -48,7 +49,8 @@ function DatasetCard({
           },
         }}
         component={Link}
-        href="#"
+        // TODO: Update URL to single dataset page
+        href={url}
```

Comment:
Isn't this URL pointing to open.africa directly?

Reply:
@kilemensi Yes, just for now. In the other PR I'm working on now, I'm updating it to the single datasets page. I could also update it to # now if need be.

Reply:
Nah, you don't need to update it here @kelvinkipruto ... Need to make sure you're working on the issue.